herd/libdir/asl-pseudocode/patches.asl

```diff
@@ -279,3 +246,62 @@ begin
     end;
 end;
 
+// =============================================================================
+
+// Optimisations: those functions are very used all over the reference, and so
+// we experience a significant slow-down if we override them.
```

Comment:
I do not understand: it seems that the functions *are* overriden just below. Isn't a negation missing?